src/SAWScript/Crucible/LLVM/Builtins.hs

```diff
@@ -1715,6 +1694,38 @@ verifyPoststate cc mspec env0 globals ret mdMap =
           fail "verifyPoststate: unexpected llvm_return specification"
         _ -> return ()
 
+getPoststateObligations ::
+  Crucible.IsSymBackend Sym bak =>
+  SharedContext ->
+  bak ->
+  IORef MetadataMap ->
+  IO [(String, MS.ConditionMetadata, Term)]
```

Comment:
It would be helpful to include some Haddocks stating what this function computes. For instance, what is the `String` in the return type? (It's hard to know without looking at the implementation.)